src/main/java/org/mskcc/utils/ParserUtil.java

```diff
@@ -0,0 +1,21 @@
+package org.mskcc.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ParserUtil {
+    private static Logger log = LoggerFactory.getLogger(ParserUtil.class);
+
+    /**
+     * Safe conversion of a string to a float with logging
+     */
+    public static Double parseDouble(String st) {
```

Comment:
Other options:
https://www.dariawan.com/tutorials/java/java-convert-string-double/

Reply:
This wraps around that method, but I want to log the "java.lang.NumberFormatException" if it happens. Will try to add a splunk alert to catch this error in the logs and send an email